aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -335,6 +335,52 @@ class QConv2dInt8 final : public c10::OperatorKernel {
     // Force output format to be NHWC
     // TODO: consider preserving input format
     // TODO: add MemoryFormat::ChannelsLast here once perf is fixed
+    auto input_scale = input_contig.q_scale();
+
+    // Re-quantizing the bias based on input scale and weight scale.
+    if (!pack_ptr.input_scale.has_value()) {
+      // Get the original weight and adjust it to uint8 from int8
+      auto weight_contig = pack_ptr.orig_weight;
+      auto bias_fp32 = pack_ptr.bias;
+      int8_t* w_data = (int8_t*)weight_contig.data_ptr<c10::qint8>();
+      Tensor qnnp_weight = at::_empty_affine_quantized(
```

Comment:
the sizes for this tensor are messed up because `pack_ptr.orig_weight` has been in 'channels_last' format. It all checks out as you never actually use it as a tensor and just as a memory buffer, but I'd add here argument MemoryFormat::ChannelsLast for sanity